### PR TITLE
changed tagline and links.

### PR DESCRIPTION
**Pull request by Ritika8081**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <p align="center">
-  <img src="https://github.com/akadeepesh/biosignal-recorder-web-private/assets/100466756/95cad4a8-8fbb-4662-afb6-414be104f69d"><br>
-   <h1 align="center">BioSignal Web Recorder</h1><hr>
+  <img src="https://github.com/Ritika8081/BioSignal-Recorder-Web/blob/main/public/steps/Chords.png"><br>
+   <h1 align="center">Chords</h1><hr>
```
Comment (akadeepesh) [inline README.md]: Changing this to relative path.

File: README.md [hunk 2]
```diff
@@ -45,9 +48,19 @@ BioSignal Web Recorder is an application based on Web Serial connection, you can
...
+
+## Contributors
+
+We are thankful to our awesome contributors, the list below is alphabetically sorted.
+
+- [Aman Maheswari](https://github.com/Amanmahe)
+- [Deepak Khatri](https://github.com/lorforlinux)
+- [Deepesh Kumar](https://github.com/akadeepesh)
+- [Mahesh Tupe](https://github.com/Asc91)
+- [Ritika Mishra](https://github.com/ritika8081)
```
Comment (akadeepesh) [inline README.md]: Will change this to github auto contributors fetch link

File: src/components/Contributors.tsx [hunk 3]
```diff
@@ -64,7 +64,9 @@ const Contributors = () => {
           <Card className="border-none -m-5">
             <CardHeader>
               <p>Chords:{VERSION}</p>
-              <CardTitle className="text-lg">Contributors</CardTitle>
+              <CardTitle className="text-lg">
+                Contributors (in alphabetical order)
+              </CardTitle>
```
Comment (akadeepesh) [inline src/components/Contributors.tsx]: Will change UI so that title don't contain (in alphabetical order) but it will be mentioned.

**Review by akadeepesh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (akadeepesh) [reply README.md]:
```
Thank you for contributing to our project! Your support is invaluable in creating & enhancing Chords-Web and making it even better. 😊

<center>
<a href="https://github.com/upsidedownlabs/Chords-Web/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=upsidedownlabs/Chords-Web" />
</a>
</center>
```

File: src/components/Contributors.tsx (hunk 3 above)
Comment (akadeepesh) [reply src/components/Contributors.tsx]: Add this div inside card title, and remove text-lg from card title
```
<div className="flex justify-between">
<span className="text-lg">Contributors</span>
<span className="text-sm text-muted-foreground">Alphabetical Order</span>
</div>
```

File: README.md (hunk 2 above)
Comment (Ritika8081) [reply README.md]: okay

**Review by akadeepesh — APPROVED** [review]

**Comment by akadeepesh** [comment]
@Ritika8081 I'm merging this now, add arduino code to a new PR

**Pull request merged**